mesonbuild/build.py

```diff
@@ -2339,10 +2344,19 @@ def process_kwargs(self, kwargs, backend):
                 # the list index of that item will not be installed
                 self.install_dir = typeslistify(kwargs['install_dir'], (str, bool))
                 self.install_mode = kwargs.get('install_mode', None)
+                # If only one tag is provided, assume all outputs have the same tag.
+                # Otherwise, we must have as much tags as outputs.
+                self.install_tag = typeslistify(kwargs.get('install_tag', [None]), (str, bool))
+                if len(self.install_tag) == 1:
+                    self.install_tag = self.install_tag * len(self.outputs)
+                if len(self.install_tag) != len(self.outputs):
```

Comment:
this could safely be elif, to avoid running the check, right?

Reply:
Yep, fixed.